relay-event-schema/src/protocol/span.rs

```diff
@@ -239,6 +239,61 @@ pub struct SentryTags {
     #[metastructure(field = "device.class")]
     pub device_class: Annotated<String>,
     // Mobile OS the transaction originated from(String).
```

Comment:
This comment got separated from its field.

Reply:
good catch!